SpringProject/src/it/java/com/gradproject2019/userConference/UserConferenceControllerIT.java

```diff
@@ -59,75 +61,120 @@ public void shouldReturn200AndSaveInterestInConference() {
 
     @Test
     public void shouldReturn401AndNotSaveWhenUserNotLoggedIn() {
-        UserConferenceRequestDto userConferenceRequestDto = userConferenceCreateRequestDto(savedUser.getId(), savedConference.getId());
-
-        ResponseEntity<UserConferenceResponseDto> response = saveInterestExpectingAuthError(baseUri, userConferenceRequestDto);
+        ResponseEntity<ErrorEntity> response = saveInterestExpectingAuthError(userConferenceRequestDto);
 
         Assert.assertEquals(401, response.getStatusCodeValue());
+        Assert.assertEquals("User unauthorized to perform action.", response.getBody().getMessage());
+    }
 
+    @Test
+    public void shouldReturn404AndNotSaveWhenConferenceDoesNotExist() {
+        userConferenceRepository.deleteAll();
```

Comment:
Similar to a previous comment, does this need to be here?

Reply:
Ditto to the previous comment.